registry/server/routerDomains/routes/getAllRouterDomains.ts

```diff
@@ -0,0 +1,16 @@
+import {
+    Request,
+    Response,
+} from 'express';
+
+import db from '../../db';
+import preProcessResponse from '../../common/services/preProcessResponse';
+
+const getAllRouterDomains = async (req: Request, res: Response): Promise<void> => {
+    const routerDomains = await db.select().from('router_domains');
+
+    res.setHeader('Content-Range', routerDomains.length); //Stub for future pagination capabilities
```

Comment:
Let's add pagination from very beginning. It's much easier to do it now.

Reply:
See example settings

Reply:
done